apps/meteor/server/services/translation/service.ts

```diff
@@ -0,0 +1,32 @@
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+import { Settings } from '@rocket.chat/models';
+import type { IUser } from '@rocket.chat/core-typings';
+
+import { ServiceClassInternal } from '../../sdk/types/ServiceClass';
+import type { ITranslationService } from '../../sdk/types/ITranslationService';
+
+export class TranslationService extends ServiceClassInternal implements ITranslationService {
+	protected name = 'translation';
+
+	private async getServerLanguage(): Promise<string> {
+		return ((await Settings.findOneById('Language'))?.value as string) || 'en';
```

Comment:
Can we cache this somewhere because if the room has 100 messages, then this will be called 100 times? Not sure if the `mem` package is now compatible with services (I hope it is :crossed_fingers: )

Reply:
This will be called once per conversation 👀 to get the message we send to users, but, caching should be good anyways so 🥳 thanks